.env.example

```diff
@@ -13,3 +13,5 @@ DB_DATABASE=
 
 JWT_SECRET=
 SALT_ROUNDS=
+
+DEBUG= #true or false
```

Comment:
why did you rename this to `DEBUG`?

Reply:
because I thought NODE_ENV shouldn't be set over .env